python/graphscope/gsctl/commands/common.py

```diff
@@ -82,5 +82,70 @@ def close():
             info("Try 'gsctl --help' for help.")
 
 
+disclaimer = """
+Disclaimer: The `estimate` command is a roughly estimation of the memory usage of the graphscope engine.
```

Comment:
`estimator`, and please disclaim it as resources estimator, rather than just for memory usage. 
It should give guide for (in the future):
- memory for gae
- disk usage for groot/interactive
- k8s config for graphscope one...